ethers-core/src/types/chain.rs

```diff
@@ -1,33 +1,36 @@
 use serde::Deserialize;
 use thiserror::Error;
 
+use core::convert::TryFrom;
```

Comment:
I believe, as of 2021 this should be imported by default?

Reply:
Hmm ok my IDE was complaining but maybe it is outdated